src/tactic/ui/tools/repo_browser_wdg.py

```diff
@@ -936,12 +936,81 @@ def get_api(my):
                 spt.app_busy.hide();
 
             }
+    
+            // Update API
+            spt.repo_browser.getElement = function(el) {
+                var repo_top = document.getElement(".spt_repo_browser_top");
+                return repo_top.getElement(el);
+            }
+
+            spt.repo_browser.dynamic_lock;
+            
+            spt.repo_browser.update_available;
+          
+            spt.repo_browser.queued_update;
         
+            spt.repo_browser.wait;
+         
+            spt.repo_browser.set_lock = function(lock) {
+                // Lock the directory listing while user is editing
+                // When lock is removed, request updates.
+                spt.repo_browser.dynamic_lock = lock;
+                
+                var dir_top = spt.repo_browser.getElement(".spt_dir_list_top");
+                
+                if (lock) {
+                    dir_top.addClass("spt_update_lock");
+                } else {
+                    dir_top.removeClass("spt_update_lock");  
+                }
+
+            }
+
+            spt.repo_browser.handle_update = function(bvr) {
```

Comment:
Good eye - thanks. I've removed this.

(At one point I was querying by expression and was going to use the returned value.)